releases/2.13.5.md

```diff
@@ -0,0 +1,47 @@
+### Highlights
+
+* TASTy reader: add support for Scala 3.0.0-RC1 (scala/scala#9501, scala/scala#9394, scala/scala#9357) @bishabosha
+* Upgrade to ASM 9.1, for JDK 16 and 17 support in the optimizer (scala/scala#9489, scala/scala#9480)
+
+### Other changes
+
+* Allow name-based extractors to be irrefutable (scala/scala#9343) @martijnhoekstra
+* Assorted improvements to exhaustivity checking in pattern matching (TODO @dwijnand will fill this in)
+* Assorted improvements to handling of higher-kinded types, aligning with Scala 3 (scala/scala#9400, scala/scala#9404, scala/scala#9405, scala/scala#9414, scala/scala#9417, scala/scala#9439) @joroKr21
+* Allow `classOf[Foo.type]` (scala/scala#9457) @Jasper-M
+* Make `-target` support JVM 13, 14, 15, 16, and 17 (scala/scala#9489, scala/scala#9481)
+* Omit `@nowarn` annotations from generated code, for forwards compatibility at compile-time (scala/scala#9491)
+* Add linting of unused context bounds (via `-Wunused:synthetics` or `-Wunused:params`) (scala/scala#9346) @som-snytt
+* Lift artificial restrictions on `ConstantAnnotation`s (scala/scala#9379)
+* Make Java `Map` wrappers handle nulls according to `put`/`remove` contract (scala/scala#9344) @som-snytt
+* Make language specification available as a PDF (scala/scala#7432) @sake92
+
+Some small changes that will ship in 2.12.14 are also included in this release.
+
+For complete 2.13.5 change lists, see [all merged PRs](https://github.com/scala/scala/pulls?q=is%3Amerged%20milestone%3A2.13.5) and [all closed bugs](https://github.com/scala/bug/issues?utf8=%E2%9C%93&q=is%3Aclosed+milestone%3A2.13.5).
+
+## Compatibility
+
+As usual for our minor releases, Scala 2.13.5 is binary-compatible with the whole Scala 2.13 series.
+
+Upgrading from 2.12? Enable `-Xmigration` while upgrading to request migration advice from the compiler.
+
+## Contributors
+
+A big thank you to everyone who's helped improve Scala by reporting bugs, improving our documentation, spreading kindness in discussions around Scala, and submitting and reviewing pull requests! You are all magnificent.
+
+This release was brought to you by 23 contributors, according to `git shortlog -sn --no-merges HEAD ^v2.13.4 ^2.12.x`. Thank you Seth Tisue, A. P. Marki, Dale Wijnand, NthPortal, Jamie Thompson, Lukas Rytz, Martijn Hoekstra, Georgi Krastev, Jason Zaugg, Jasper Moeys, Sakib Hadziavdic, Anatolii Kmetiuk, Arnaud Gourlay, Marcono1234, Chia-Ping Tsai, Mike Skells, Stefan Zeiger, Waleed Khan, Yann Bolliger, Guillaume Martres, 梦境迷离, Ethan Atkins, Da.
```

Comment:
Is `Da` a person?

Reply:
it's Darcy Shen, @da-tubi. I'll fix it and I'll also add a mailmap entry so it happens automatically next time.

Reply:
https://github.com/scala/scala/pull/9511